code/datalab-app/src/api/schema/dataStorage.js

```diff
@@ -0,0 +1,62 @@
+import {
+  GraphQLEnumType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from 'graphql';
+import dataStorageRepository from '../dataaccess/dataStorageRepository';
+
+const StorageType = new GraphQLEnumType({
+  name: 'StorageType',
+  description: 'Data store classes within DataLabs',
+  values: {
+    nfs: {
+      description: 'Network File System (NFS) share.',
+      value: 1,
+    },
+  },
+});
+
+const DataStoreType = new GraphQLObjectType({
+  name: 'DataStore',
+  description: 'DataLabs data store type.',
+  fields: {
+    id: {
+      type: GraphQLInt,
+    },
+    name: {
+      type: GraphQLString,
+    },
+    storageType: {
+      type: StorageType,
+    },
+    capacityTotal: {
+      type: GraphQLInt,
+    },
+    capacityUsed: {
+      type: GraphQLInt,
+    },
+    linkToStorage: {
+      type: GraphQLString,
+    },
+  },
+});
+
+export const dataStorage = {
+  description: 'List of currently provisioned DataLabs data storage.',
+  type: new GraphQLList(DataStoreType),
+  resolve: () => dataStorageRepository.getAll(),
+};
+
+export const dataStore = {
+  description: 'Details of a single Datalabs data store.',
```

Comment:
Docs should probably be more descriptive and include information about the parameters